src/Avalonia.Controls/TreeViewItem.cs

```diff
@@ -65,6 +73,15 @@ public bool IsSelected
             set { SetValue(IsSelectedProperty, value); }
         }
 
+        /// <summary>
```

Comment:
Nit: should be just `Gets the depth of the item.` as the setter is private.

Reply:
Ah yes, that is correct, will fix that...